### PR TITLE
Updated autopilot guide

### PR DESCRIPTION
**Pull request by kaitlincarter-hc**
Updated the autopilot guide to prepare for the migration to the learn. The updates will be pushed to consul.io before being moved to learn.

### CODE REVIEW

**Comment by kaitlincarter-hc** [comment]
What's new:
* Updated introduction
* New summary
* code snippets for each section, using the cli
* server health checking moved to end

What's removed:
* The read scaling section

Review/feedback needed: 
* How do we feel about removing the read scaling section? 
* Does the guide have one consistent voice/tone?
* Are there any issues with flow?

File: website/source/docs/guides/autopilot.html.md [hunk 1]
```diff
@@ -19,28 +19,48 @@ the Agent configuration must be set to 3 or higher on all servers. In Consul
 information, see the [Version Upgrade section](/docs/upgrade-specific.html#raft_protocol)
 on Raft Protocol versions.
 
-## Configuration
+In this guide we will learn more about Autopilot's features.
+
+* Dead server cleanup
+* Server Stableization
+* Redundcy zone tags
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: Typos, should be "Stabilization" and "Redundancy"

File: website/source/docs/guides/autopilot.html.md [hunk 2]
```diff
@@ -19,28 +19,48 @@ the Agent configuration must be set to 3 or higher on all servers. In Consul
...
+
+Note, in this guide we are using  examples from a Consul 1.4 cluster, we
+are starting with Autopilot enabled by default.
+
+## Default Configuration
 
 The configuration of Autopilot is loaded by the leader from the agent's
 [Autopilot settings](/docs/agent/options.html#autopilot) when initially
-bootstrapping the cluster:
+bootstrapping the cluster. Since Autopilot and it's feartures are already
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: Typo, should be "features"

File: website/source/docs/guides/autopilot.html.md [hunk 3]
```diff
@@ -19,28 +19,48 @@ the Agent configuration must be set to 3 or higher on all servers. In Consul
...
     "server_stabilization_time": "10s",
-    "redundancy_zone_tag": "az",
+    "redundancy_zone_tag": "",
     "disable_upgrade_migration": false,
     "upgrade_version_tag": ""
 }
 ```
 
+All Consul servers should have Autopilot and its features either enabled
+or disabled to ensure consistency accross servers encase of a failure. Additionally,
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: "encase" > "in case"

File: website/source/docs/guides/autopilot.html.md [hunk 4]
```diff
@@ -19,28 +19,48 @@ the Agent configuration must be set to 3 or higher on all servers. In Consul
...
+Autopilot must be enabled to use any of the features, but the features themselves
+can be configured independently. Meaning you can enable or disable any of the features
+seperately, at any time.
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: "seperately" > "separately"

File: website/source/docs/guides/autopilot.html.md [hunk 5]
```diff
@@ -65,34 +107,143 @@ DisableUpgradeMigration = false
...
+CleanupDeadServers = false
+LastContactThreshold = 200ms
+MaxTrailingLogs = 250
+ServerStabilizationTime = 5s
+RedundancyZoneTag = ""
+DisableUpgradeMigration = false
+UpgradeVersionTag = ""
+```
+
+Now we have disabled dead server cleanup and set the server stabalization time to 5 seconds.
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: "stabalization" > "stabilization"

File: website/source/docs/guides/autopilot.html.md [hunk 6]
```diff
@@ -65,34 +107,143 @@ DisableUpgradeMigration = false
...
+CleanupDeadServers = false
+LastContactThreshold = 200ms
+MaxTrailingLogs = 250
+ServerStabilizationTime = 5s
+RedundancyZoneTag = "uswest1"
+DisableUpgradeMigration = false
+UpgradeVersionTag = ""
+```
+
+For our Autopilot features, we now have disabled dead server cleanup, server stabalization time to 5 seconds, and
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: "stabalization" > "stabilization"

File: website/source/docs/guides/autopilot.html.md [hunk 7]
```diff
@@ -136,83 +287,13 @@ $ curl localhost:8500/v1/operator/autopilot/health
...
-
-If the `UpgradeVersionTag` setting is set, Consul will use its value to look for a
-version in each server's specified [`-node-meta`](/docs/agent/options.html#_node_meta)
-tag. For example, if `UpgradeVersionTag` is set to `build`, and `-node-meta build:0.0.2`
-is used when starting a server, that server's version will be `0.0.2` when considered in
-a migration. The upgrade logic will follow semantic versioning and the version string
-must be in the form of either `X`, `X.Y`, or `X.Y.Z`.
+To learn more about the Autopilot settings we did not configure,
+[last_contact_threshold](https://www.consul.io/docs/agent/options.html#last_contact_threshold)
+and [max_trailing_logs](https://www.consul.io/docs/agent/options.html#max_trailing_logs)
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: Missing a comma after "and max_trailing_logs"

File: website/source/docs/guides/autopilot.html.md [hunk 8]
```diff
@@ -136,83 +287,13 @@ $ curl localhost:8500/v1/operator/autopilot/health
...
+either read the agent confituration documenation or use the help flag with the
+opertor autopilot `consul operator autopilot set-config -h`.
```
Comment (kyhavlov) [inline website/source/docs/guides/autopilot.html.md]: "confituration" > "configuration"
"opertor" > "operator"

**Pull request merged**